pkg/granted/request/close.go

```diff
@@ -0,0 +1,150 @@
+package request
+
+import (
+	"context"
+	"fmt"
+
+	"connectrpc.com/connect"
+	"github.com/common-fate/cli/printdiags"
+	"github.com/common-fate/clio"
+	"github.com/common-fate/grab"
+	"github.com/common-fate/granted/pkg/cfaws"
+	"github.com/common-fate/granted/pkg/cfcfg"
+	"github.com/common-fate/sdk/config"
+	"github.com/common-fate/sdk/eid"
+	accessv1alpha1 "github.com/common-fate/sdk/gen/commonfate/access/v1alpha1"
+	"github.com/common-fate/sdk/service/access/grants"
+	"github.com/common-fate/sdk/service/access/request"
+	identitysvc "github.com/common-fate/sdk/service/identity"
+	"github.com/urfave/cli/v2"
+)
+
+var closeCommand = cli.Command{
+	Name:  "close",
+	Usage: "Close an active Just-In-Time access to a particular entitlement",
+	Flags: []cli.Flag{
+		&cli.StringFlag{Name: "aws-profile", Required: false, Usage: "Close a JIT access for a particular AWS profile"},
```

Comment:
Lets reduce this to just 'profile' which aligns with other commands
```suggestion
		&cli.StringFlag{Name: "profile", Required: false, Usage: "Close a JIT access for a particular AWS profile"},
```